test/fixtures/inline-import-equality.js

```diff
@@ -0,0 +1,9 @@
+import exposed from '../../src/cjs.js?exposes=myGlobal!./global-commonjs2-single-export.js';
+
+import imported from './global-commonjs2-single-export.js'
+
+export default {
+  exposedEqualsGlobal: exposed === myGlobal,
+  importedEqualsGlobal: imported === myGlobal,
+  exposedEqualsImported: exposed === imported,
+}
```

Comment:
Created a new test fixture to ensure correct behavior, instead of just testing for the presence of the reference.

All three should be `true`. If the reference is missing, the last two will be false, due to different loader options resulting in multiple instances of the same module.

The snapshot test of the result will ensure changes to this behavior will be caught in the future.